app/filters.js

```diff
@@ -4,9 +4,12 @@ const filters = {
   date: Vue.filter('date', (date) => {
     // undef first load
     if (!date) return;
+    // Purpose: remove the time zone.
+    // Converts date into a string, splits the string at the spaces, splice
+    // keeps items 0-5 (removes the time zone), rejoins at the spaces into a
```

Comment:
Why have `remove time zone` twice? You get the purpose when you state the purpose, so why include it here?

Reply:
We should be extremely clear about what the purpose is. Thus, listing it as the purpose. Then in the description of what's actually happening, we list where the purpose is fulfilled.

Reply:
Fair enough, dude.

Reply:
Thanks, bro.